ca/ca.go

```diff
@@ -460,18 +431,16 @@ func (ca *certificateAuthorityImpl) issuePrecertificateInner(ctx context.Context
 		return nil, nil, berrors.InternalServerError("failed to prepare precertificate signing: %s", err)
 	}
 
-	if features.Enabled(features.StoreLintingCertificateInsteadOfPrecertificate) {
-		nowNanos := ca.clk.Now().UnixNano()
-		_, err = ca.sa.AddPrecertificate(context.Background(), &sapb.AddCertificateRequest{
-			Der:          lintCertBytes,
-			RegID:        issueReq.RegistrationID,
-			Issued:       nowNanos,
-			IssuerNameID: int64(issuer.Cert.NameID()),
-			OcspNotReady: true,
-		})
-		if err != nil {
-			return nil, nil, err
-		}
+	nowNanos := ca.clk.Now().UnixNano()
+	_, err = ca.sa.AddPrecertificate(context.Background(), &sapb.AddCertificateRequest{
+		Der:          lintCertBytes,
+		RegID:        issueReq.RegistrationID,
+		Issued:       nowNanos,
```

Comment:
```suggestion
	_, err = ca.sa.AddPrecertificate(context.Background(), &sapb.AddCertificateRequest{
		Der:          lintCertBytes,
		RegID:        issueReq.RegistrationID,
		Issued:       ca.clk.Now().UnixNano(),
```

Reply:
I'm going to pass on this suggestion to keep this a pure deprecation PR and not add side tweaks. Thanks!